recipes/umi-transfer/meta.yaml

```diff
@@ -11,20 +11,16 @@ source:
   
 build:
   number: 0
+  skip: true
```

Comment:
This disables the build of the recipe.

Reply:
Oops. It shows that I have no bloody clue of `conda`, and that I have just been copy and pasting stuff from other Rust recipes around.